samples/HttpClientFactorySample/Program.cs

```diff
@@ -23,7 +26,16 @@ public class Program
                     c.DefaultRequestHeaders.Add("Accept", "application/vnd.github.v3+json"); // Github API versioning
                     c.DefaultRequestHeaders.Add("User-Agent", "HttpClientFactory-Sample"); // Github requires a user-agent
                 })
-                .AddLogging()
+
+                // Retry requests to github using our retry handler
+                .AddHttpMessageHandlerPolicy("github", DefaultStages.Reliability, b => b.AdditionalHandlers.Add(new RetryHandler()))
```

Comment:
Adding a policy in the `Reliability` stage means that this lambda will run before our 'logging' policy. 

Then our logging policy will add a handler at the beginning and end to make a delicious logging sandwich 🍞